CS_Ed_Week/code-examples/filter_under_number.py

```diff
@@ -0,0 +1,8 @@
+def Filter_Under_Number(input_number, input_list):
+  temp_list = []
+  for x in range(len(input_list)):
+    if input_list[x] < input_number:
+      temp_list.append(input_list[x])
+  return temp_list
```

Comment:
Another possible solution to this problem would be:

```python
for x in input_list:
    if x < input_number:
    temp_list.append(x)
```
This approach is a bit more concise because it doesn't use list subscripting/indexing. Instead, it assigns the actual value of the current element to `x`. Of course, your solution has the advantage of using some more of the features of Python, but it all depends on what your goal was :)